.github/workflows/ci.yml

```diff
@@ -303,6 +305,14 @@ jobs:
         PYTHON:
           - {VERSION: "3.7", NOXSESSION: "tests-nocoverage"}
           - {VERSION: "3.13", NOXSESSION: "tests"}
+          - {VERSION: "pypy-3.10", NOXSESSION: "tests-nocoverage"}
+          - {VERSION: "pypy-3.11", NOXSESSION: "tests-nocoverage"}
+        exclude:
+          # We need to exclude the below configuration because there is no 32-bit pypy3
+          - WINDOWS: {ARCH: 'x86', WINDOWS: 'win32'}
+            PYTHON: {VERSION: "pypy-3.10", NOXSESSION: "tests-nocoverage"}
+          - WINDOWS: {ARCH: 'x86', WINDOWS: 'win32'}
+            PYTHON: {VERSION: "pypy-3.11", NOXSESSION: "tests-nocoverage"}
```

Comment:
In general, backport branches should match what's on main for whatever they're backporting. In this case we don't test pypy on macOS and Windows, so that shouldn't be included here.

Reply:
Got it! I submitted a new patch!